Makefile

```diff
@@ -1,12 +1,15 @@
-.PHONY: ui vendor build ui-agent ui-manager container-agent container-agent-push container-manager container-manager-push release-container-agent-multiversions push
+.PHONY: ui vendor build ui-agent ui-manager container-agent container-agent-push container-manager container-manager-push release-container-agent-multiversions push container-frontend-auth container-frontend-auth-push container-frontend-noauth container-frontend-noauth-push
 
 CONTAINER_TAG ?= tsidentity/tornjak-spire-server:latest
+CONTAINER_TAG_FRONTEND ?= tsidentity/tornjak-ui:latest
 CONTAINER_VERSION_IMAGEPATH ?= tsidentity/tornjak-spire-server
 CONTAINER_VERSION_GHCR_IMAGEPATH ?= ghcr.io/spiffe/tornjak-spire-server
```

Comment:
we should re-do this for GHCR (these are the automated images created by GitHub workflow) as well.